CHANGELOG-core.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add new env variable ETH_SECONDARY_URL. Default is unset. You may optionally set this to an http(s) ethereum RPC client URL. If set, transactions will also be broadcast to this secondary ethereum node. This allows transaction broadcasting to be more robust in the face of primary ethereum node bugs or failures.
+- At startup, the chainlink node will create a new "rescue" address. This will initially be used to pay for cancelling stuck transactions.
```

Comment:
Is it called a funding address?

Reply:
Yes

Reply:
Should we change the wording in the release notes?